src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -12,7 +12,7 @@
             [murphy :refer [try!]]
             [puppetlabs.puppetdb.query-eng :as qeng]
             [puppetlabs.puppetdb.query.monitor :as qmon]
-            [puppetlabs.trapperkeeper.services.webserver.jetty9 :as jetty9]
+            [puppetlabs.trapperkeeper.services.webserver.jetty10 :as jetty10]
```

Comment:
Both requires can be removed.  It was just used for the name spacing, which doesn't exist in jetty10.

Reply:
Removed this one, which is the other?